server/platform/services/searchengine/interface.go

```diff
@@ -7,6 +7,7 @@ import (
 	"time"
 
 	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/mattermost/mattermost/server/public/shared/request"
 )
 
 type SearchEngineInterface interface {
```

Comment:
At some point, I would like to come back to the interface and migrate the rest of the methods to use a `request.CTX`. But that can happen later.